hr/api_views.py

```diff
@@ -62,3 +63,14 @@ def post(self, request, *args, **kwargs):
             return Response({'salary': salary})
 
         return Response(serializer.errors, status=400)
+
+
+class DepartmentViewSet(viewsets.ModelViewSet):
+    queryset = Department.objects.all()
+    serializer_class = DepartmentSerializer
+
+    @action(detail=True, methods=['get'])
+    def employee_count(self):
+        department = self.get_object()
+        employee_count = department.employee_set.count()
```

Comment:
```suggestion
        employee_count = Employee.objects.filter(position__department_id=pk).count()
```
and for detail view you need pk in arguments